doc/Command-Reference.md

```diff
@@ -6564,6 +6568,301 @@ This command is used to set the counter polling interval. Default is 20 seconds.
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#sflow)
 
+## SNMP
+
+### SNMP Show commands
+
+**show run snmp**
+
+This command displays the global SNMP configuration that includes the location, contact, community, and user setttings.
+
+- Usage:
+  """
+  show run snmp
+  """
```

Comment:
Code blocks use three backticks (`), not three double-quotes. Please replace all throughout and verify the rendered markdown looks correct.

Reply:
fixed.